pom.xml

```diff
@@ -9,34 +9,32 @@
   <groupId>org.nuxeo.ecm.platform</groupId>
   <artifactId>marketplace-shibboleth-authentication</artifactId>
   <version>2.6.0-SNAPSHOT</version>
-  <packaging>zip</packaging>
-  <name>Shibboleth Authentication</name>
+  <packaging>pom</packaging>
+  <name>Utility POM for Shibboleth Authentication Package</name>
 
-  <dependencies>
-    <!-- Additional bundles to be deployed by the MP -->
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-platform-login-shibboleth</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-usermapper</artifactId>
-    </dependency>
-  </dependencies>
+  <modules>
+    <module>marketplace</module>
+  </modules>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.nuxeo.build</groupId>
-        <artifactId>ant-assembly-maven-plugin</artifactId>
-        <configuration>
-          <buildFiles>
-            <buildFile>${basedir}/src/main/assemble/assembly.xml</buildFile>
-          </buildFiles>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <profiles>
+    <profile>
+      <id>ftest</id>
+      <modules>
+        <module>ftest/webdriver</module>
```

Comment:
Might be put in the default modules declaration.

Reply:
Will do when enabling the tests back.